### DISCUSSION TITLE
How do I change the taskbar to black colour?

### DISCUSSION THREAD

**Discussion by djefferd** [valinet/ExplorerPatcher#3082] [Q&A] [answered]
I am on Win 11, Couldn't move my taskbar to the left side. Installed ExplorerPatcher, successfully moved taskbar to the left side. Hooray. However, the taskbar is light grey, and I want it to be near-black (dark mode style). How do I accomplish this? Thanks in advance.

**Comment by pyrates999** [comment, marked as answer]
https://support.microsoft.com/en-us/windows/how-to-use-the-taskbar-in-windows-0657a50f-0cc7-dbfd-ae6b-05020b195b07#WindowsVersion=Windows_11

**Comment by djefferd** [comment]
Thank you. The trick was changing "Choose your mode" in that System > Personalization menu.
